### ISSUE TITLE
incanter.charts relies on an apparently outdated jfreechart, which deprecated some stuff

### ISSUE THREAD

**Issue by joinr** [incanter/incanter#389]
org.jfree.chart.ChartUtilities class appears deprecated, thinking it exists in ChartUtils.  Other problems may exist.

**Comment by joinr** [comment]
Looks like they made a massive jump from 1.019 to 1.5 (no intermediate releases.....1.5 popped out Nov 2017).  Unclear what's deprecated.

**Comment by joinr** [comment]
Lots of info [here](https://github.com/jfree/jfreechart/issues/66) as well as a somewhat weak migration guide on the main github repo readme.

**Comment by alexott** [comment]
Yes, I tried to switch to latest version, but this requires investment of the significant amount of time to do it...

**Comment by MawiraIke** [comment]
Looks like this is already fixed with no issues.
